packages/get-platform/src/getPlatform.ts

```diff
@@ -151,60 +156,75 @@ type GetOpenSSLVersionParams = {
  * On Linux, returns the OpenSSL version excluding the patch version, e.g. "1.1.x".
  * Reading the version from the libssl.so file is more reliable than reading it from the openssl binary.
  * This function never throws.
+ *
+ * TODO: we should probably validate the output of this function, and ensure it's contained in `supportedLibSSLVersions`
```

Comment:
Is that a TODO for now or later?

Reply:
For https://github.com/prisma/prisma/issues/17154